cora/util/hputil.py

```diff
@@ -1,66 +1,9 @@
 """
-============================================
-Healpix Utilities (:mod:`~cora.util.hputil`)
-============================================
-
 Convenience functions for dealing with Healpix maps.
 
 Makes heavy use of the `healpy`_ module.
 
 .. _`healpy`: http://github.com/healpy
-
```

Comment:
Why remove the categorisation of the various methods?

Reply:
They are already automatically listed. If you list them again in these categories, they appear twice.

Reply:
Why is that though? It didn't used to be the case. Has some setting changed?

Where something like this is present (i.e. someone has attempted to write better organised docs), we should really try and use it.

Reply:
I think a lot had changed since the docs were built the last time. This is how I managed to make it build without having to list every single class by hand in the docstrings, so I applied the same recipe to all repositories. I think a lost categorization is a small price to pay for having the full API reference with little effort. Is it so important?

Reply:
Aaaargh. I should have read the other PRs more closely, I didn't appreciate that this has happened everywhere.

It matters because we're destroying the useful part of the documentation. An actual human went through all these methods, figured out what they were, and grouped them into useful categories to make it easier for other people to understand what they do and find what they need. We shouldn't throw that away lightly because it's a lot of effort to recreate.

Also, the html documentation isn't only thing the docstrings are for. I never look at the html docs, and possibly never will, but I read the documentation directly in the source files often. So it's not clear that goal justifies removing everything.

Anyway, I apologise for not catching this sooner, but I don't think we can go down this route. It seems like a pretty big regression to me. Can it be configured on a module by module basis?

As for the annoyance of generating all the autosummary entries, there must be ways of making the listings far easier. I imagine there's ways of making sphinx do this for you, but even if not in a script you could open the file, and get a list of all module-level names with `__doc__` entries, and massage it into something that works that you can insert either by hand or programatically. We can even have a linter in the actions that makes sure that any new top level symbol gets mentioned in the docs.

Reply:
AFAICT the change came in a commit after my last review of the caput PR, so I think that's why it wasn't spotted earlier.

Reply:
It looks like we could just use https://sphinx-automodapi.readthedocs.io/en/latest/ for modules which don't have their own custom documentation. Would that solve the issues you were having @nritsche ?

Reply:
Listed separately as references now